flexget/plugins/clients/pyload.py

```diff
@@ -155,11 +155,17 @@ def add_entries(self, task, config):
 
         for entry in task.accepted:
             # bunch of urls now going to check
-            content = entry.get('description', '') + ' ' + quote(entry['url'])
-            content = json.dumps(content)
+            contents = []
+            description = entry.get('description', '')
+            if description != '':
+                contents.append(description)
+            contents.append(quote(entry['url']))
+            content = " ".join(contents)
+
+            content = repr(content)
```

Comment:
repr is used to get the object creation code in python, very likely not what you wanted here. Perhaps you wanted to cast string? But even that is unnecessary since content is guaranteed to be a string already.

Please rethink / explain the repr usage here and in general

Reply:
pyload is using ast.literal_eval to parse the arguments, so this is absolutely on purpose ;-)

Reply:
well, that's a good explanation for sure :D